src/Workspaces/Core/Desktop/Workspace/Storage/AbstractPersistentStorageService.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ErrorReporting;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Microsoft.CodeAnalysis.SolutionSize;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Storage
+{
+    /// <summary>
+    /// A service that enables storing and retrieving of information associated with solutions,
+    /// projects or documents across runtime sessions.
+    /// </summary>
+    internal abstract partial class AbstractPersistentStorageService : IPersistentStorageService2
```

Comment:
I renamed the file to match; GitHub isn't necessarily picking up the rename.

Reply:
can you undo the rename?

Reply:
@CyrusNajmabadi I broke the rename out into a separate commit. Not great but will make things a bit easier I hope.

Reply:
still seems awful on github.